sprokit/processes/kwiver_type_traits.h

```diff
@@ -115,6 +115,8 @@ create_port_trait( gsd, gsd, "GSD for image in meters per pixel." );
 create_port_trait( image, image, "Single frame image." );
 create_port_trait( left_image, image, "Single frame left image." );
 create_port_trait( right_image, image, "Single frame right image." );
+create_port_trait( image1, image, "Single frame first image." );
+create_port_trait( image2, image, "Single frame second image." );
```

Comment:
These two port traits are algorithm specific enough to be moved to the process source.

Reply:
Moved.